# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/service/VRService.cpp
# Commit: 898ed02804fe
# Full Hash: 898ed02804fefc9786112f135b9621cc444e6860
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-23 15:52:15
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r e1b7abc99ae9 -r 898ed02804fe gfx/vr/service/VRService.cpp
--- a/gfx/vr/service/VRService.cpp	Wed May 22 12:44:04 2019 +0000
+++ b/gfx/vr/service/VRService.cpp	Wed May 22 22:27:37 2019 +0000
@@ -52,7 +52,7 @@
 already_AddRefed<VRService> VRService::Create() {
   MOZ_ASSERT(NS_IsMainThread());
 
-  if (!gfxPrefs::VRServiceEnabled()) {
+  if (!StaticPrefs::VRServiceEnabled()) {
     return nullptr;
   }
 
@@ -73,7 +73,7 @@
 #if defined(XP_WIN)
       mMutex(NULL),
 #endif
-      mVRProcessEnabled(gfxPrefs::VRProcessEnabled()) {
+      mVRProcessEnabled(StaticPrefs::VRProcessEnabled()) {
   // When we have the VR process, we map the memory
   // of mAPIShmem from GPU process.
   // If we don't have the VR process, we will instantiate